reference-implementation/test/readable-byte-stream.js

```diff
@@ -1,25 +1,13 @@
 const test = require('tape-catch');
 
 test('ReadableByteStream can be constructed with no errors', t => {
-  new ReadableByteStream();
-
-  t.doesNotThrow(() => new ReadableByteStream(), 'ReadableByteStream constructed with no parameters');
-  t.doesNotThrow(() => new ReadableByteStream({ }),
-                 'ReadableByteStream constructed with an empty object as parameter');
-  t.doesNotThrow(() => new ReadableByteStream(undefined),
-                 'ReadableByteStream constructed with undefined as parameter');
-  let x;
-  t.doesNotThrow(() => new ReadableByteStream(x),
-                 'ReadableByteStream constructed with an undefined variable as parameter');
+  t.doesNotThrow(() => new ReadableStream({ pullInto() {} }),
+                 'ReadableByteStream constructed with an empty underlying byte source object as parameter');
   t.end();
 });
 
 test('ReadableByteStream: getReader(), then releaseLock()', t => {
-  const rbs = new ReadableByteStream({
-    pull() {
```

Comment:
Should we maybe set the default HWM to 0 if a pullInto is supplied? Always calling pull asking for one byte seems kind of silly.